cnames_active.js

```diff
@@ -2634,6 +2634,7 @@ var cnames_active = {
   "wdd": "wangduanduan.github.io",
   "weather": "phanendraguptha.github.io/weather",
   "weaver": "maxkfranz.github.io/weaver", // noCF? (don´t add this in a new PR)
+  "web.blooket": "redyetidev.github/blooket.js-web",
```

Comment:
```suggestion
  "web.blooket": "redyetidev.github/blooket.js-web", // noCF
```
👋 As you're requesting a sub-sub-domain, the Cloudflare proxy and universal SSL cannot be enabled. You'll need to add the `// noCF` comment to the end of your entry to reflect that the proxy is disabled.